python_modules/libraries/dagster-airflow/dagster_airflow/factory.py

```diff
@@ -240,14 +241,15 @@ def _make_airflow_dag(
 
 def make_airflow_dag(
     module_name,
-    pipeline_name,
+    job_name,
     run_config=None,
     mode=None,
     instance=None,
     dag_id=None,
     dag_description=None,
     dag_kwargs=None,
     op_kwargs=None,
+    pipeline_name=None,
 ):
     """Construct an Airflow DAG corresponding to a given Dagster pipeline.
```

Comment:
change this to job?

Reply:
did you mean changing "job_name" to "job"

Reply:
I mean "Construct an Airflow DAG corresponding to a given Dagster job"

Reply:
ah misread the github highlighting.. roger